### PR TITLE
[systemtest] Add test for KafkaStatus certificate

### PR DESCRIPTION
**Pull request by im-konge**
### Type of change

- Bugfix
- Enhancement / new feature

### Description

Add new asserts that checks if certs from KafkaStatus and secret certs are matching (both external and internal TLS).
Also add test that checks if certs are present in KafkaStatus and if they are matching with secret certs.

Should fix tests from ListenersST (failed in nightly jobs)

Testing: https://issues.redhat.com/browse/ENTMQST-1103

### Checklist

- [x] Write tests
- [x] Make sure all tests pass

### CODE REVIEW

**Comment by lkral13** [comment]
@strimzi-ci run tests profile=regression testcase=ListenersST

**Comment by lkral13** [comment]
@strimzi-ci run tests profile=regression testcase=ListenersST

**Comment by lkral13** [comment]
All tests are green on local machine and on openstack - I tested testCustomChainCertificatesForNodePort for 5 times - always green

**Comment by lkral13** [comment]
@strimzi-ci run tests profile=regression testcase=ListenersST

**Comment by lkral13** [comment]
@strimzi-ci run tests profile=regression testcase=ListenersST

**Comment by Frawless** [comment]
@ppatierno @scholzj  could you guys take a look on it? We should merge it today

File: systemtest/src/test/java/io/strimzi/systemtest/kafka/ListenersST.java [hunk 1]
```diff
@@ -372,6 +374,17 @@ void testCustomCertNodePortAndTlsRollingUpdate() throws Exception {
 
         KafkaUser aliceUser = KafkaUserResource.tlsUser(CLUSTER_NAME, userName).done();
 
+        String externalCerts = getKafkaStatusCertificates("external", NAMESPACE, CLUSTER_NAME);
+        String externalSecretCerts = getKafkaSecretCertificates(CLUSTER_NAME + "-cluster-ca-cert", "ca.crt");
+
+        String internalCerts = getKafkaStatusCertificates("tls", NAMESPACE, CLUSTER_NAME);
+        String internalSecretCerts = getKafkaSecretCertificates(CLUSTER_NAME + "-cluster-ca-cert", "ca.crt");
```
Comment (ppatierno) [inline systemtest/src/test/java/io/strimzi/systemtest/kafka/ListenersST.java]: why reading the CA cert twice?

Comment (im-konge) [reply systemtest/src/test/java/io/strimzi/systemtest/kafka/ListenersST.java]: okay fixed :) thanks

**Comment by Frawless** [comment]
Cherry-pick to `release-0.17.x` please

**Pull request merged**